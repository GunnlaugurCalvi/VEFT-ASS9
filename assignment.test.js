const {add, throws} = require("./index");
//import * as th from './index';

describe('add', () =>{
	test('Should successfully add two integers together', () => {
		expect(add(1,1)).toBe(2);
	});
	test('should return just 1', () => {
		expect(add(null, 1)).toBe(1);
	});
	test('nothing is entered returns 0', () => {
		expect(add()).toEqual(NaN);
	});
});
describe('throws', () => {
	test('Should not throw error', () => {
		const callSum = () => {
			throws(true);
		};
	expect(callSum).toThrowError('You need to mock me');
	});
/*	test('should test if it returns false', () => {
		throws(true,() => {
			expect(true).toThrowError('You need to mock me');
		});	
	});	*/
});
