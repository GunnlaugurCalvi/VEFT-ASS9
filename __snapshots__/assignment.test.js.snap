// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should create 5 employees as a lawyer when its created 1`] = `
Object {
  "data": Array [
    Object {
      "__v": 0,
      "_id": "59ee7fdb50f4eb1a3c17e71b",
      "jobTitles": Array [
        "laywer 0",
      ],
      "name": "Billie Joe JimBob",
    },
    Object {
      "__v": 0,
      "_id": "59ee7fdb50f4eb1a3c17e71c",
      "jobTitles": Array [
        "laywer 1",
      ],
      "name": "Billie Joe JimBob",
    },
    Object {
      "__v": 0,
      "_id": "59ee7fdb50f4eb1a3c17e71d",
      "jobTitles": Array [
        "laywer 2",
      ],
      "name": "Billie Joe JimBob",
    },
    Object {
      "__v": 0,
      "_id": "59ee7fdb50f4eb1a3c17e71e",
      "jobTitles": Array [
        "laywer 3",
      ],
      "name": "Billie Joe JimBob",
    },
    Object {
      "__v": 0,
      "_id": "59ee7fdb50f4eb1a3c17e71f",
      "jobTitles": Array [
        "laywer 4",
      ],
      "name": "Billie Joe JimBob",
    },
  ],
}
`;
